### ISSUE TITLE
Failing with - 'Command update_rc is missing' while configuring zookeeper.

### ISSUE THREAD

**Issue by JoeBmc** [deric/puppet-zookeeper#62]
Hi Deric,

Starting the zookeeper service fails with the below error.
````
==> centos-6.6: Error: /Stage[main]/Zookeeper::Service/Service[zookeeper-server]/ensure: change from stopped to running failed: Command update_rc is missing
```
I think, `update_rc` is not used with centos-6. Please correct me if I am wrong.
```
Enviroment
 -> Vagrant 1.8.1
-> Virtualbox 5.0.14
-> ISO image - puppetlabs/centos-6.6-64-puppet

Below are the configuration used
Vagrantfile
                     puppet module install puppetlabs-stdlib;
                     puppet module install richardc-datacat;
                     puppet module install deric-zookeeper;
```
puppet file
```puppet
exec { 'update-rpm-packages':
    command => '/usr/bin/yum update -y',
    timeout => 1800
  }

  package {'epel-release':
    ensure => 'installed',
    require => Exec['update-rpm-packages'],
    before => Class['java']
  }

  class { 'java':
    distribution => 'jre',
    before => Class['zookeeper'],
    require => Exec['update-rpm-packages']
  }

  # Install Zookeeper at 127.0.0.1
  class { 'zookeeper':
    repo => 'cloudera',
    cdhver => '5',
    packages => ['zookeeper', 'zookeeper-server'],
    service_name => 'zookeeper-server',
    initialize_datastore => true,
    client_ip => $::ipaddress_lo,
    require => Exec['update-rpm-packages']
  }
```

Full puppet logs for zookeeper.
```
==> centos-6.6: Notice: /Stage[main]/Main/Node[centos]/Exec[update-rpm-packages]/returns: executed successfully
==> centos-6.6: Notice: /Stage[main]/Main/Node[centos]/Package[epel-release]/ensure: created
==> centos-6.6: Notice: /Stage[main]/Java/Package[java]/ensure: created
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Os::Redhat/Package[zookeeper]/ensure: created
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Os::Redhat/Package[zookeeper-server]/ensure: created
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Post_install/User[zookeeper]/comment: comment changed 'ZooKeeper' to 'Zookeeper'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/etc/zookeeper/conf]/ensure: ensure changed 'link' to 'directory'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/etc/zookeeper/conf/environment]/ensure: defined content as '{md5}d03659a53bb0c3c67aa62bc4640ece6b'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/etc/zookeeper/conf/log4j.properties]/ensure: defined content as '{md5}45a4cc016b7400b596b3321f8f0466cd'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/etc/zookeeper/conf/zoo.cfg]/ensure: defined content as '{md5}ffdecec87c01dc3a1ef433ebb77f9a1d'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/etc/zookeeper/conf/myid]/ensure: defined content as '{md5}b026324c6904b2a9cb4b88d6d61c81d1'
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/Exec[initialize_datastore]/returns: executed successfully
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Config/File[/var/lib/zookeeper/myid]/ensure: ensure changed 'file' to 'link'
==> centos-6.6: Error: Command update_rc is missing
==> centos-6.6: Error: /Stage[main]/Zookeeper::Service/Service[zookeeper-server]/ensure: change from stopped to running failed: Command update_rc is missing
==> centos-6.6: Notice: /Stage[main]/Zookeeper::Service/Service[zookeeper-server]: Triggered 'refresh' from 1 events
==> centos-6.6: Notice: /Stage[main]/Zookeeper/Anchor[zookeeper::end]: Dependency Service[zookeeper-server] has failures: true
==> centos-6.6: Warning: /Stage[main]/Zookeeper/Anchor[zookeeper::end]: Skipping because of failed dependencies
==> centos-6.6: Notice: Finished catalog run in 481.43 seconds
```

**Comment by deric** [comment]
Which version of `puppet-zookeeper` do you use? I guess this error was already fixed master branch #61. I'll try to draft new release.

**Comment by deric** [comment]
Should be fixed in [v0.5.4](https://github.com/deric/puppet-zookeeper/releases/tag/v0.5.4).

**Comment by JoeBmc** [comment]
The quick fix worked fine. 
Thanks for your help and appreciate your efforts.
